kyo-prelude/shared/src/test/scala/kyo2/AbortTest.scala

```diff
@@ -67,10 +69,8 @@ class AbortsTest extends Test:
             }
             "in handle" in {
                 val effect: Int < Abort[String | Boolean] =
-                    kyo2.Abort.fail("failure")
-                val handled: Result[String | Boolean, Int] =
-                    kyo2.Abort.run[String | Boolean](effect).eval
-                assert(handled == Result.error("failure"))
+                    kyo2.Abort.error("failure")
+                assertDoesNotCompile("kyo2.Abort.run[String | Boolean](effect)")
```

Comment:
This scenario wasn't correct and I've added a `Tag` to make it fail at compile time. The `ClassTag` was inferred to `Any`, effectively handling any `Abort` suspensions.